### ISSUE TITLE
Crash, codec problem, and more features request.

### ISSUE THREAD

**Issue by kevinsmia1939** [halverneus/org.kde.video#3]
I use KDE neon 5.10 
KDE plasma version 5.10.2
KDE Framework 5.35.0
Qt version 5.9.0
Linux kernel 4.8.0-56-generic

Crash
The video wallpaper crash when I open something, sometime it happen, where do I look for the error log? (I have not experience crash yet with new update, surprising since it was like 2-3 days ago)

Codec
I think it can only run .webm, I tried mp4 and mk4 it just black screen (but with sound), I tried to convert to .webm and it works. (With lastest update this some how fix, I will try more)

New features request
Make playlist option to choose what next video to play next.
Loop playlist

**Comment by halverneus** [comment]
Thanks for the feature requests. It'll probably be a while, but I'll see what it takes to implement logging and a playlist. As codecs go, it doesn't ship with any codecs; it is entirely dependent on codecs installed on your system. I recommend installing 'kubuntu-restricted-extras' to gather codecs for playing other file types.

**Comment by Hobart2967** [comment]
Looks like it is accepting m3u already, isnt it?

**Comment by Hobart2967** [comment]
Yes, Support for m3u works like a charm. Just create a video playlist vlc and choose "all files" as file type. Then you can select it in filepicker and it works
